src/main/java/Todos.java

```diff
@@ -0,0 +1,22 @@
+public class Todos {
```

Comment:
Please consider naming class in PascalCase ie "Todos" to "ToDos" to distinguish between the 2 words